### ISSUE TITLE
Refactoring `Site Settings` in admin dashboard

### ISSUE THREAD

**Issue by ajaxtown** [letterpad/letterpad#34]
There is too much of stuff in site setting and it will be nice to refactor this. 

<img width="500" alt="screen shot 2018-06-22 at 19 11 54" src="https://user-images.githubusercontent.com/1502352/41789597-2bf701f8-7650-11e8-92ee-cfa94187e051.png">

One of the idea will be to have tabs:

<img width="321" alt="screen shot 2018-06-22 at 19 14 10" src="https://user-images.githubusercontent.com/1502352/41789707-7f6fa8ee-7650-11e8-9ec6-36bbdffbd64d.png">

Or accordion:
<img width="321" alt="screen shot 2018-06-22 at 19 16 14" src="https://user-images.githubusercontent.com/1502352/41789786-c719e704-7650-11e8-8eed-4fb2ee441369.png">

Or we can also create more items on the left sidebar.

What would be a good approach to solve this ?

**Comment by alder** [comment]
I think tabs is a good idea.

**Comment by ajaxtown** [comment]
Thanks @tienpham94 for taking this up. I am always available for quries. So let me know if you need anything.

**Comment by ajaxtown** [comment]
Tabs is good but if we have too many it wont display well in mobile view. Or maybe in mobile we turn them into icons ?

**Comment by tienpham94** [comment]
Yeah I can make an MVP real quick and we can see what are the options
